sdk/src/wallet/account/operations/reissue.rs

```diff
@@ -81,21 +81,25 @@ where
 
                 // Check inclusion state for each attachment
                 let block_ids_len = block_ids.len();
-                let mut conflicting = false;
-                for (index, block_id_) in block_ids.clone().iter().enumerate() {
-                    let block_metadata = self.client().get_block_metadata(block_id_).await?;
-                    if let Some(inclusion_state) = block_metadata.ledger_inclusion_state {
-                        match inclusion_state {
-                            LedgerInclusionState::Included | LedgerInclusionState::NoTransaction => {
-                                return Ok(*block_id_);
-                            }
-                            // only set it as conflicting here and don't return, because another reissued block could
+                let mut failed = false;
+                for (index, block_id) in block_ids.clone().iter().enumerate() {
+                    let block_metadata = self.client().get_block_metadata(block_id).await?;
+                    if let Some(transaction_state) = block_metadata.tx_state {
+                        match transaction_state {
+                            TransactionState::Finalized => return Ok(*block_id),
```

Comment:
Do we want this to accept Confirmed or Finalized?

Reply:
Don't know yet, added a TODO